app/presenters/finder_presenter.rb

```diff
@@ -186,6 +186,16 @@ def all_content_finder?
     self.content_item['content_id'] == 'dd395436-9b40-41f3-8157-740a453ac972'
   end
 
+  def topic_finder?
+    values.include?('parent') && topic_finder_parent.present?
+  end
+
+  def topic_finder_parent
+    registry = Services.registries.all['full_topic_taxonomy']
+    parent = registry.find_parent_by_id values['parent']
+    parent.nil? ? '' : parent
```

Comment:
I'd return `parent` here. If it's nil then calling `present?` on it will return false anyway.

Reply:
👍